src/main/scala/com.felixmulder.dotty.plugin/DottyPlugin.scala

```diff
@@ -22,37 +24,78 @@ object DottyPlugin extends AutoPlugin {
       println(s"Latest Dotty nightly build version: $latest")
```

Comment:
Weren't you the one to say not to use println with sbt? Or maybe that was only in `build.sbt`...